Src/libCZI/utilities.h

```diff
@@ -204,3 +204,74 @@ struct ParseEnumHelper
     }
 };
 
+/// This class allows to calculate the area covered by a set of rectangles. The mode of operation
+/// is:
+/// - Create an instance of the class and add the rectangles to it (using AddRectangle) .
+/// - Then, for a given rectangle, call CalcAreaOfIntersectionWithRectangle in order to get the area of the intersection  
+///    of this rectangle with the union of the rectangles added before.
+/// The rectangles being added do not have to follow any order, or are required to be non-overlapping.
```

Comment:
You could hint that it will usually help performance to add in order of decreasing size.

Reply:
done